cmd/deploy.go

```diff
@@ -83,6 +84,11 @@ func newDeployCmd() *cobra.Command {
 	return deployCmd
 }
 
+// NormalizeAzureRegion returns a normalized Azure region with whilte spaces removed and converted to lower case
```

Comment:
typo in the comment `whilte` --> `white`

Reply:
We could add this function to https://github.com/Azure/acs-engine/blob/master/pkg/helpers/helpers.go and re-use it here and in pkg/api